artemis/amqp-connector/src/main/java/org/apache/activemq/artemis/integration/amqp/AMQPConnectorServiceFactory.java

```diff
@@ -50,6 +50,8 @@
       properties.add(TransportConstants.KEYSTORE_PASSWORD_PROP_NAME);
       properties.add(TransportConstants.TRUSTSTORE_PATH_PROP_NAME);
       properties.add(TransportConstants.TRUSTSTORE_PASSWORD_PROP_NAME);
+      properties.add(TransportConstants.USE_GLOBAL_WORKER_POOL_PROP_NAME);
+      properties.add(TransportConstants.REMOTING_THREADS_PROPNAME);
```

Comment:
What does this change do?

Reply:
This allows those settings to be set in the broker XML and override defaults.